CONTRIBUTING.md

```diff
@@ -83,6 +79,14 @@ you've contributed contains language that is friendly and accessible.
 If you're unsure about any of the above, feel free to ask us for
 clarification via the mailing list.
 
+# Git Best Practices
```

Comment:
Moved here from the 'Coding Standards' doc because it's more about PR workflow than anything to do with the code.